app/src/main/java/com/infomaniak/mail/data/cache/mailboxInfo/MailboxController.kt

```diff
@@ -196,11 +197,11 @@ class MailboxController @Inject constructor(
         }
 
         private fun getInvalidPasswordMailboxesQuery(userId: Int, realm: TypedRealm): RealmQuery<Mailbox> {
-            return realm.query("${checkHasUserId(userId)} AND NOT ($hasValidPassword OR $isMailboxLocked)")
+            return realm.query("${checkHasUserId(userId)} AND NOT ($hasValidPassword) AND NOT ($isLocked) AND $isValidInLDAP")
```

Comment:
Do you have to have parenthesis for `AND NOT ($isLocked)` ? `AND NOT $isLocked` is not enough ? Same question for `getValidMailboxesQuery`.

Reply:
True.
I removed most of them.